### ISSUE TITLE
読みたい論文メモ

### ISSUE THREAD

**Issue by mts1104** [mts1104/read-papers#2]
- CNN系の有名どころ
- 物体検出の有名どころ
- GANの原著

**Comment by mts1104** [comment]
Random Erasing Data Augmentation
<https://arxiv.org/abs/1708.04896>

Improved Regularization of Convolutional Neural Networks with Cutout
<https://arxiv.org/abs/1708.04552>

**Comment by mts1104** [comment]
Batch Normalization

**Comment by mts1104** [comment]
StyleGAN
